controllers/ListItemController.php

```diff
@@ -17,6 +17,34 @@
 
 	class ListItemController
 	{
+		/**
+			* Swagger doc block
+			*
+			* @OA\Get(
+			* path="ListItem/get_list_items",

```

Comment:
This path here refers to the API endpoint path which in this case would be: `/list/item/{listID}`

Reply:
made the change.